### DISCUSSION TITLE
Add in support for Google Find my device network?

### DISCUSSION THREAD

**Discussion by Cassander313** [malmeloo/FindMy.py#30] [Ideas]
Love what you all have built!  I was wondering if you think it would be possible to add in the ability for us have include the google find my devices as well?

**Comment by malmeloo** [comment]
This would definitely be interesting to implement! I'd kill to have a tool and tag that could use both networks at once...

There has been some research in https://github.com/seemoo-lab/openhaystack/discussions/210, definitely something to keep an eye on.

**Comment by malmeloo** [comment]
In case you missed it, there has been quite some progress here: https://github.com/leonboe1/GoogleFindMyTools

To limit the scope of the library, I am not currently planning on adding support for Google's network to FindMy.py.
